stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiChatApi.kt

```diff
@@ -177,25 +178,50 @@ internal class MoshiChatApi @Suppress("LongParameterList") constructor(
         return configApi.getAppSettings().map(AppSettingsResponse::toDomain)
     }
 
-    override fun sendMessage(channelType: String, channelId: String, message: Message): Call<Message> {
+    override fun sendMessage(
+        channelType: String,
+        channelId: String,
+        message: Message,
+        skipPushNotification: Boolean,
+        skipEnrichUrl: Boolean,
+    ): Call<Message> {
```

Comment:
As far as we are including the new flags to the `Message` class, can't we use them from inside the argument and avoid adding extra flags to the method params?

Reply:
Updated